package.json

```diff
@@ -2,7 +2,7 @@
     "name": "aws-toolkit-vscode",
     "displayName": "AWS Toolkit",
     "description": "Amazon Web Services toolkit for browsing and updating cloud resources",
-    "version": "1.30.0-SNAPSHOT",
+    "version": "1.29.0",
```

Comment:
CHANGE THIS BACK PRIOR TO MERGE!!!!!! This edit shouldn't be necessary for c9 testing.

Reply:
This PR isn't intended to be merged

Reply:
This PR will never be merged.   A v1.29 with Step Functions enabled was requested for testing purposes, so it's not up to date with master.